### ISSUE TITLE
Add an additional final step: Save permalink

### ISSUE THREAD

**Issue by jidanni** [hholzgra/maposmatic#69]
Perhaps, to save disk space on the server, add an additional final step: Save permalink.

I.e., only create a permalink like
https://maposmatic.osm-baustelle.de/results/2021/12/27/197936_2021-12-27_16-58_dan-s-map.jpg
(or even additional short version too)
if the user is happy with the results, and wishes to save a permalink too them.
Else don't put
https://maposmatic.osm-baustelle.de/results/2021/12/27/197936_2021-12-27_16-58_dan-s-map.jpg
in the URL bar, and don't keep any info from this session on the server.

**Comment by hholzgra** [comment]
I had thought about adding a user management system at some point, where registered users can have features like permanent storage, higher render priority, maps that are not publicly visible, etc. ... but so far never followed through on this.

Given that rendered files are usually available for at least two months, and can easily be re-generated that was never really an issue yet, but I understand that it would be if you'd like to generate printable maps for public use that should be accessible for a longer time frame.

**Comment by jidanni** [comment]
Yes currently apparently some links are just generated sequentially,
allowing some users to guess other users' maps, and thus discover buried
treasure locations...

But anyway however long a link is supposed to last always needs to be
mentioned on the same page with the link.
